components/BaseHeader.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <header>
+    <div class="logo">
+      <img src="/images/logo-rms.svg" alt="RMS Logo" />
+      <p>OWL <br /><span>Support Center</span></p>
+    </div>
+    <BaseHeaderNavigation />
+    <div>
+      <p>Icons go here</p>
+    </div>
+  </header>
+</template>
+
+<style lang="scss" scoped>
```

Comment:
two style sections

Reply:
@tomgalpin Good catch. No clue how this happened!